website/project/metadata/schemas.py

```diff
@@ -1,17 +1,11 @@
 import os
 import json
 
+
 def ensure_schema_structure(schema):
```

Comment:
Ideally the solution would be to perform schema validation of the JSON files, and not allow malformed files into the pipeline- if this function is needed, something has arguably gone wrong.

In the interests of time I won't push for this change, but....